### DISCUSSION TITLE
Hackathon: 2024-04-02

### DISCUSSION THREAD

**Discussion by stefaniebutland** [earthaccess-dev/earthaccess#507] [Hack days]
Reporting out on earthaccess hack days.

Please create a new comment (at the very bottom, use the text box with a button that says "comment", not "reply") to write about what you did today!

Please include links to GitHub issues or pull requests relevant to your work, and use those issues or pull requests as the "source of truth" for documenting the work; this makes future readers jobs easier :) It's OK to duplicate some information while writing a summary here!

Use the "reply" feature to have a discussion under any given comment.

**Comment by chuckwondo** [comment]
I created a new issue regarding erroneous/missing type hints: [Incorrect type hints for CMR queries](https://github.com/nsidc/earthaccess/issues/508)

**Comment by itcarroll** [comment]
Introduced nasa/python_cmr#29 and nasa/python_cmr#30. Which will hopefully lead to closing #486 if the contributions to python_cmr go smoothly!

**Comment by mfisher87** [comment]
I briefly explored running integration tests locally on a screen share, then @nikki-t and I debugged an integration test together in support of https://github.com/nsidc/earthaccess/issues/447

**Reply by nikki-t** [reply]
I created #509 to support the integration test work we did so that we make the test behavior around EDL credentials clearer.

**Reply by mfisher87** [reply]
Thanks so much for following through on that! If it was left to me, we might be waiting longer than we like :rofl:

**Comment by jules32** [comment]
@ateucher and I focused on R users and earthaccess. 

We tested the [Cookbook How Do I section for R](https://nasa-openscapes.github.io/earthdata-cloud-cookbook/how-tos/find-data/find-r.html), which uses the reticulate package. This needs some updating and next steps, including with CMR parsers in R - Andy will share more in an issue and have something to share with R users on April 17 with the Earthdata Cloud Clinic

**Reply by ateucher** [reply]
I think this can be done a little more cleanly without using reticulate and earthaccess. Issue [here](https://github.com/NASA-Openscapes/earthdata-cloud-cookbook/issues/318).

**Reply by jules32** [reply]
Of course, yes! thanks Andy
